vcs/git/repo.go

```diff
@@ -104,13 +104,13 @@ func (r *Repository) ResolveTag(name string) (vcs.CommitID, error) {
 	return "", vcs.ErrTagNotFound
 }
 
-func (r *Repository) Branches() ([]*vcs.Branch, error) {
+func (r *Repository) Branches(_ vcs.BranchesOptions) (branches []*vcs.Branch, total uint, err error) {
```

Comment:
You can omit the `_`. It doesn't matter, but I didn't realize that was possible until a few months ago myself and thought I'd mention it in case you didn't know.